docs/docs/guides/web3_plugin_guide/index.md

```diff
@@ -0,0 +1,86 @@
+---
+sidebar_position: 1
```

Comment:
```suggestion
sidebar_position: 0
```

Reply:
I don't think the plugin guide should be positioned above the migration guide, if anything the tree shaking guide should have it's index updated to `2`

Reply:
[This](https://github.com/web3/web3.js/pull/5558/commits/e99b645b0a5971f404f61c11b79feeddd408a631) commit updates the tree shaking guide to position `2`

Reply:
Interesting this sequence numbers are taking effects at guides level in rendering. I suggested these sequence numbers for inside plugin guide.  

For sequence at guides level I think following makes more sense:
1. Migration Guide
2. Plugin guide
3. Tree Shaking

Reply:
@spacesailor24 this PR looks good, except ordering and sub ordering guides changes.

Reply:
This is the `index` file for `web3_plugin_guide` directory, so it's a top-level list item like migration guide and tree shaking. It's the other files, `plugin_authors` and `plugin_users` that will be nested underneath "Plugin Guide" which is why their indexes are `0` and `1` respectively

Reply:
@jdevcs This is the `index` file for `web3_plugin_guide` directory, so it's a top-level list item like migration guide and tree shaking. It's the other files, `plugin_authors` and `plugin_users` that will be nested underneath "Plugin Guide" which is why their indexes are `0` and `1` respectively